### PR TITLE
Clean up JSPJava7Test test ordering problem and cleanup other JSPJava tests

### LINKED ISSUE #28508: Clean up JSPJava7Test test ordering problem and cleanup other JSPJava tests
1. If the tests in JSPJava7Test are run in a different order on different JDKs one stopServer call does not have an allowed set of warnings like the other servers do. Update this to ensure there are no test ordering issues.
2. JSPJava7Test - wait for config updates

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #28508

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#libby

File: dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPJava11Test.java [hunk 1]
```diff
@@ -109,8 +108,16 @@ public void testJava11viaUseJDKCompiler() throws Exception {
         LOG.info("New server configuration used: " + configuration);
         server.setMarkToEndOfLog();
         server.updateServerConfiguration(configuration);
+
+        server.setMarkToEndOfLog();
+        server.updateServerConfiguration(configuration);
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPJava11Test.java]: Need to fix this no need to update twice

**Comment by pnicolucci** [comment]
#build

**Review by isaacrivriv — APPROVED** [review]
LGTM, thanks!

**Comment by pnicolucci** [comment]
#libby

**Pull request merged**